README.md

```diff
@@ -25,37 +32,13 @@ the `--ruby_out` option). It does not generate Twirp services and clients; that
 
 ### Install the `protoc-gen-twirp_ruby` plugin
  
-Install the gem by adding it to your Gemfile:
+Run `gem install protoc-gen-twirp_ruby` or add it to your Gemfile:
```

Comment:
Probably worth keeping the note that this is intended as a development dependency, not a runtime one.